tests/test_csv.py

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
for tests:

- you **dont** need this.
- we **do** need a test_version.py, you should be able to base it on the other tests and just test to make sure that the store.get_version() method is being called once.